src/lib/utilities/stores.js

```diff
@@ -63,6 +63,16 @@ function createClothesStore() {
             return itemsOfType;
         },
 
+        getClothingItemsByFilters: (filterList) => {
+            let itemsOfFilters;
+            update(items => {
+                itemsOfFilters = items.filter((item) => {item.filters.cozy === filterList.cozy && item.filters.formal === filterList.formal
+                                && item.filters.temphigh === filterList.temphigh && item.filters.templow === filterList.templow; });
+                return items;
+            });
+            return itemsOfFilters;
+        },
+
```

Comment:
I think the way we wanted to handle the temp high and low was to just pass in a temp so filterList.temp and then return outfits that have a temp high above that and temp low below it.

Reply:
Also maybe change the name of filter list to filterQuery or something, because technically it isn't a list, it is just an object that has `cozy: boolean, formal: boolean, temp: number`

Reply:
When you create the filter popover you might just want to have some predefined options for temperature that map to number you pass in as temp. i.e if the user selects "cold" you would set temp to 40 or something. LMK if that makes sense.